rn-cli.config.js

```diff
@@ -15,7 +15,7 @@ const config = {
   getBlacklistRE() {
     return blacklist([
       /extensions\/.*\/server\/.*/,
-      /scripts\/.*/
+      /\/scripts\/.*/
```

Comment:
I think that it should be `/(\\|\/)scripts/.*/` because Windows could have \ as the path separator.